.github/workflows/runtests.yml

```diff
@@ -151,16 +140,17 @@ jobs:
 
     runs-on: ubuntu-20.04
     strategy:
+      fail-fast: false
       matrix:
-        split: ['1/2', '2/2']
+        group: [1, 2, 3, 4]
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
-      - run: wget -qO- https://github.com/pyodide/pyodide/releases/download/0.20.1a1/pyodide-build-0.20.1a1.tar.bz2 | tar xjf -
+          node-version: 18.x
+      - run: wget -qO- https://github.com/pyodide/pyodide/releases/download/0.21.3/pyodide-build-0.21.3.tar.bz2 | tar xjf -
       - run: python3 setup.py bdist_wheel
-      - run: node bin/test_pyodide.mjs --split=${{ matrix.split }} 2>/dev/null  # ignore node exception
+      - run: node bin/test_pyodide.mjs --group=${{ matrix.group }} --splits=4 2/dev/null  # ignore node exception
```

Comment:
This should be `2>/dev/null`.

Reply:
Sadly doesn't seem to have resolved the hanging on completion issue.